### ISSUE TITLE
labeling French data

### ISSUE THREAD

**Issue by RamziRahli** [CristianViorelPopa/BART-TL-topic-label-generation#3]
Hello, 

First of all thank you for the work you have done.

I have some questions about it: 

Can this model be applied for the French language? 

If the answer is yes, what are the modifications I have to do? and what kind of data I need ?

I want to do the labeling on the results of my LDA clustering, do I have to run the process from the beginning?

Thanks in advance.

Cordialy.

**Comment by CristianViorelPopa** [comment]
Hello @RamziRahli !

The models can't be applied for French directly, they were only trained and fine-tuned for English. Unfortunately, I think you have to start from scratch. 😔

You would need a set of documents in French and go through this notebook: https://github.com/CristianViorelPopa/BART-TL-topic-label-generation/blob/main/notebooks/end_to_end_workflow.ipynb. You would need to be careful about the structure of the data, since the scripts here parse XML files. Change the scripts in the `lda` directory if you need to.

For the pre-trained model, instead of the `bart-large` that is used here, try a model that's pre-trained on French data, such as this: https://huggingface.co/sshleifer/tiny-mbart, since it's also a BART variant.

Hope this helped. Let me know if I can help you with other questions!
